app/Http/routes.php

```diff
@@ -168,20 +154,90 @@
         return view('dashboard.pages.edit_episode');
     });
 
-    Route::get('/channels', function () {
-        return view('dashboard.pages.view_channels');
-    });
-
-    Route::get('/channel/create', function () {
-        return view('dashboard.pages.create_channel');
-    });
+/*
+/-------------------------------------------------------------------------------
+/ Admin User
+/-------------------------------------------------------------------------------
+*/
+    Route::get('/users', [
+        'uses' => 'UserController@index',
+        'as'   => '/users'
+    ]);
+
+    Route::get('/user/edit/{id}', [
```

Comment:
**Code Quality - Nitpick**

It should be `/user/{id}/edit` , that's more RESTFUL